packages/authgear-core/src/types.ts

```diff
@@ -223,3 +226,70 @@ export interface _OIDCTokenResponse {
   expires_in: number;
   refresh_token?: string;
 }
+
+/**
+ * Possible values: "LoggedIn" | "NoSession" | "Unknown"
+ *
+ * The state of the user session in authgear. An authgear instance is in one and only one specific
+ * state at any given point in time.
+ *
+ * For example, the session state of an authgear instance that is just constructed always has
+ * "Unknown". After a call to [Authgear.configure], the session state would be
+ * "LoggedIn" if a previous session is found, or "NoSession" if there is
+ * no such sessions.
+ *
+ * @public
+ */
+export type SessionState = "LoggedIn" | "NoSession" | "Unknown";
```

Comment:
I suppose using UPPER_SNAKE_CASE matches the JS convention on constant values better.